pkg/ee/cluster-backup/resources/user-cluster/cluster_backup.go

```diff
@@ -43,12 +43,28 @@ const (
 	clusterRoleBindingName = "velero"
 	clusterBackupAppName   = "velero"
 	defaultBSLName         = "default-cluster-backup-bsl"
+
+	cloudCredentialsSecretName           = "velero-cloud-credentials"
+	defaultCloudCredentialsSecretKeyName = "cloud"
+
+	version       = "v1.12.0"
+	pluginVersion = "v1.2.1"
 )
 
 // NamespaceReconciler creates the namespace for velero related resources on the user cluster.
 func NamespaceReconciler() reconciling.NamedNamespaceReconcilerFactory {
 	return func() (string, reconciling.NamespaceReconciler) {
 		return resources.ClusterBackupNamespaceName, func(ns *corev1.Namespace) (*corev1.Namespace, error) {
+			ns.Labels = map[string]string{
+				"kubernetes.io/metadata.name": "velero",
+
+				"pod-security.kubernetes.io/audit":           "privileged",
+				"pod-security.kubernetes.io/audit-version":   "latest",
+				"pod-security.kubernetes.io/enforce":         "privileged",
+				"pod-security.kubernetes.io/enforce-version": "latest",
+				"pod-security.kubernetes.io/warn":            "privileged",
+				"pod-security.kubernetes.io/warn-version":    "latest",
```

Comment:
Why is this added here?

Reply:
The labels are generated by the `velero install` command. I added them for compatibility with upstream configuration.

Reply:
Okay, if this is from generated commands, can this be documented as a comment so people know how to generate this information for newer velero versions?

Reply:
done. Good catch!